### PR TITLE
zh-translation: /news/2019/announcing-1.4/

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1208
ref: https://github.com/servicemesher/istio-official-translation/issues/1209
ref: https://github.com/servicemesher/istio-official-translation/issues/1210
ref: https://github.com/servicemesher/istio-official-translation/issues/1211

### CODE REVIEW

File: content/zh/news/2019/announcing-1.4/_index.md [hunk 1]
```diff
@@ -1,15 +1,56 @@
...
 publishdate: 2019-11-14
-attribution: The Istio Team
 release: 1.4.0
 skip_list: true
 aliases:
     - /zh/news/announcing-1.4
 ---
 
-We are pleased to announce the release of Istio 1.4!
+我们很高兴的宣布，Istio 1.4 发布！
```
Comment (ExBs2724) [inline content/zh/news/2019/announcing-1.4/_index.md]: 很高兴的宣布 => 很高兴地宣布

File: content/zh/news/2019/announcing-1.4/_index.md [hunk 2]
```diff
@@ -1,15 +1,56 @@
...
 
 {{< relnote >}}
+
+Istio 1.4 继续努力改善 Istio 的用户体验，着重于简化。
```
Comment (ExBs2724) [inline content/zh/news/2019/announcing-1.4/_index.md]: 感觉"着重于简化" 后面少了宾语。
听上去好奇怪

File: content/zh/news/2019/announcing-1.4/_index.md [hunk 3]
```diff
@@ -1,15 +1,56 @@
...
+这也将取代旧的 [v1alpha1 RBAC 策略](/zh/docs/reference/config/security/istio.rbac.v1alpha1/)。
+
+## 改进的故障排除{#improved-troubleshooting}
+
+我们引入了 [`istioctl analysis`](/zh/docs/ops/diagnostic-tools/istioctl-analyze/) 命令，以改进对网格的故障排除。
+它可以帮您检查网格中的配置问题，甚至在将新配置提交到网格之前对其进行验证。
+
+## 更好的 sidecar{#better-sidecar}
+
+我们一直在做大量工作来改进Envoy，包括其功能集和使用体验。
```
Comment (ExBs2724) [inline content/zh/news/2019/announcing-1.4/_index.md]: 没有加 空格

File: content/zh/news/2019/announcing-1.4/_index.md [hunk 4]
```diff
@@ -1,15 +1,56 @@
...
+现在，Envoy在崩溃时可以更优雅地退出，支持更多指标，并且可以镜像一定比例的流量。
```
Comment (ExBs2724) [inline content/zh/news/2019/announcing-1.4/_index.md]: 空格

**Comment by ExBs2724** [comment]
这个是目录结构发生变化了吗😂

**Comment by ilylia** [comment]
是的，新增了好几篇blog

**Pull request merged**